AU: Vista de login-#15

jorandmol opened the pull request:
He completado la vista de login y la he integrado con la funcionalidad de inicio de sesión de los usuarios (relativa a la tarea #6).

Esta perfectamente adaptada para dispositivos móviles y he añadido un espacio emergente cuando el formulario es erróneo.

Closes #15

Commit:
Añadida la vista de login de los usuarios - #15

He creado la vista de login de usuario, además de haber generado una hoja css con los estilos. Únicamente faltaría dar algunos retoques y corregir la visualización en pantallas grandes.

Commit:
Integrando la rama AU-#15 con la develop

Commit:
Correcciones e integración con el base - #15

He ampliado el logo, suavizado el fondo y he integrado la interfaz con la base. Faltaría añadir la capa del backend.

Commit:
Añadida funcionalidad a la vista de login - #15

He añadido la capa de backend a la vista de login sustituyendo el html provisional por el desarrollado por mi. A este le he introducido un espacio para mostrar la alerta en caso de error en el formulario.

fragonlug reviewed (changes requested):
Los cambios resultantes del frontend se han integrado correctamente con el backend, por mi parte decir que se debería subir el archivo de migrations para la correcta ejecución de las pruebas, junto a resolver una incidencia indicada por [Codacy](https://app.codacy.com/gh/youRoom-ISPP/youRoom/pullRequest?prid=7134624).

mrf1989 reviewed (changes requested):
@jorandmol De igual manera que te comenta @fragonlug, considero que deberías hacer pull de lo que te falta para conseguir que Travis te pase con éxito y revisar Codacy también.

Commit:
Correcciones en estilo - #15

He corregido el estilo para que el formulario se muestre bien centrado. Además, añado la migración que evita el fallo en Travis más una corrección para Codacy.

fragonlug reviewed (commented):
Se ha acepta el merge.

jorandmol reviewed (commented):
Hice los cambios para que pase Travis y, al intentar solucionar lo de Codacy, aparece un error sin sentido provenientes de la develop. De todas formas pensamos que es un error ignorable.
